### ISSUE TITLE
Possible incompatibility with better animals plus

### ISSUE THREAD

**Issue by rfwainug** [Laike-Endaril/Dynamic-Stealth#20]
---- Minecraft Crash Report ----

WARNING: coremods are present:
  PassableLeavesCore (PassableLeavesCore-1.12.2-1.0.0.jar)
  NWRTweak (redstonepaste-mc1.12-1.7.5.jar)
  LucraftCoreCoreMod (LucraftCore-1.12.2-2.3.4.jar)
  CreativePatchingLoader (CreativeCore_v1.9.37_mc1.12.2.jar)
  EmoticonsCore (emoticons-0.2.1-1.12.2.jar)
  HCASM (HammerCore-1.12.2-2.0.4.1.jar)
Contact their authors BEFORE contacting forge

// I just don't know what went wrong :(

Time: 2/2/19 4:42 AM
Description: Watching Server

java.lang.Error: ServerHangWatchdog detected that a single server tick took 60.00 seconds (should be max 0.05)
	at net.minecraft.world.chunk.BlockStateContainer.func_186013_a(BlockStateContainer.java:84)
	at net.minecraft.world.chunk.storage.ExtendedBlockStorage.func_177484_a(ExtendedBlockStorage.java:63)
	at net.minecraft.world.chunk.Chunk.<init>(Chunk.java:120)
	at net.minecraft.world.gen.ChunkGeneratorOverworld.func_185932_a(ChunkGeneratorOverworld.java:252)
	at net.minecraft.world.gen.ChunkProviderServer.func_186025_d(ChunkProviderServer.java:143)
	at net.minecraft.world.World.func_72964_e(World.java:310)
	at net.minecraft.world.ChunkCache.<init>(ChunkCache.java:37)
	at net.minecraft.pathfinding.PathNavigate.func_179680_a(SourceFile:113)
	at net.minecraft.pathfinding.PathNavigateGround.func_179680_a(SourceFile:51)
	at net.minecraft.pathfinding.PathNavigateGround.func_75494_a(SourceFile:73)
	at net.minecraft.entity.ai.EntityAIAttackMelee.func_75250_a(EntityAIAttackMelee.java:63)
	at com.fantasticsource.dynamicstealth.server.ai.NPEAttackTargetTaskHolder.func_75250_a(NPEAttackTargetTaskHolder.java:22)
	at net.minecraft.entity.ai.EntityAITasks.func_75774_a(SourceFile:94)
	at net.minecraft.entity.EntityLiving.func_70626_be(EntityLiving.java:763)
	at net.minecraft.entity.EntityLivingBase.func_70636_d(EntityLivingBase.java:2352)
	at net.minecraft.entity.EntityLiving.func_70636_d(EntityLiving.java:577)
	at net.minecraft.entity.EntityAgeable.func_70636_d(EntityAgeable.java:178)
	at net.minecraft.entity.passive.EntityAnimal.func_70636_d(SourceFile:43)
	at its_meow.betteranimalsplus.common.entity.EntityFeralWolf.func_70636_d(EntityFeralWolf.java:281)
	at net.minecraft.entity.EntityLivingBase.func_70071_h_(EntityLivingBase.java:2172)
	at net.minecraft.entity.EntityLiving.func_70071_h_(EntityLiving.java:295)
	at its_meow.betteranimalsplus.common.entity.EntityFeralWolf.func_70071_h_(EntityFeralWolf.java:297)
	at net.minecraft.world.World.func_72866_a(World.java:1993)
	at net.minecraft.world.WorldServer.func_72866_a(WorldServer.java:832)
	at net.minecraft.world.World.func_72870_g(World.java:1955)
	at net.minecraft.world.World.func_72939_s(World.java:1759)
	at net.minecraft.world.WorldServer.func_72939_s(WorldServer.java:613)
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:767)
	at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:397)
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:668)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:526)
	at java.lang.Thread.run(Thread.java:748)

A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Server Watchdog
Stacktrace:
	at net.minecraft.world.chunk.BlockStateContainer.func_186013_a(BlockStateContainer.java:84)
	at net.minecraft.world.chunk.storage.ExtendedBlockStorage.func_177484_a(ExtendedBlockStorage.java:63)
	at net.minecraft.world.chunk.Chunk.<init>(Chunk.java:120)
	at net.minecraft.world.gen.ChunkGeneratorOverworld.func_185932_a(ChunkGeneratorOverworld.java:252)
	at net.minecraft.world.gen.ChunkProviderServer.func_186025_d(ChunkProviderServer.java:143)
	at net.minecraft.world.World.func_72964_e(World.java:310)
	at net.minecraft.world.ChunkCache.<init>(ChunkCache.java:37)
	at net.minecraft.pathfinding.PathNavigate.func_179680_a(SourceFile:113)
	at net.minecraft.pathfinding.PathNavigateGround.func_179680_a(SourceFile:51)
	at net.minecraft.pathfinding.PathNavigateGround.func_75494_a(SourceFile:73)
	at net.minecraft.entity.ai.EntityAIAttackMelee.func_75250_a(EntityAIAttackMelee.java:63)
	at com.fantasticsource.dynamicstealth.server.ai.NPEAttackTargetTaskHolder.func_75250_a(NPEAttackTargetTaskHolder.java:22)
	at net.minecraft.entity.ai.EntityAITasks.func_75774_a(SourceFile:94)
	at net.minecraft.entity.EntityLiving.func_70626_be(EntityLiving.java:763)
	at net.minecraft.entity.EntityLivingBase.func_70636_d(EntityLivingBase.java:2352)
	at net.minecraft.entity.EntityLiving.func_70636_d(EntityLiving.java:577)
	at net.minecraft.entity.EntityAgeable.func_70636_d(EntityAgeable.java:178)
	at net.minecraft.entity.passive.EntityAnimal.func_70636_d(SourceFile:43)
	at its_meow.betteranimalsplus.common.entity.EntityFeralWolf.func_70636_d(EntityFeralWolf.java:281)
	at net.minecraft.entity.EntityLivingBase.func_70071_h_(EntityLivingBase.java:2172)
	at net.minecraft.entity.EntityLiving.func_70071_h_(EntityLiving.java:295)
	at its_meow.betteranimalsplus.common.entity.EntityFeralWolf.func_70071_h_(EntityFeralWolf.java:297)
	at net.minecraft.world.World.func_72866_a(World.java:1993)
	at net.minecraft.world.WorldServer.func_72866_a(WorldServer.java:832)
	at net.minecraft.world.World.func_72870_g(World.java:1955)
	at net.minecraft.world.World.func_72939_s(World.java:1759)
	at net.minecraft.world.WorldServer.func_72939_s(WorldServer.java:613)
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:767)
	at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:397)
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:668)

-- Thread Dump --
Details:
	Threads: "Downloader 3" Id=78 TIMED_WAITING on java.util.concurrent.SynchronousQueue$TransferStack@70298d52
	at sun.misc.Unsafe.park(Native Method)
	-  waiting on java.util.concurrent.SynchronousQueue$TransferStack@70298d52
	at java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:215)
	at java.util.concurrent.SynchronousQueue$TransferStack.awaitFulfill(SynchronousQueue.java:460)
	at java.util.concurrent.SynchronousQueue$TransferStack.transfer(SynchronousQueue.java:362)
	at java.util.concurrent.SynchronousQueue.poll(SynchronousQueue.java:941)
	at java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1073)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	...

"Downloader 2" Id=77 TIMED_WAITING on java.util.concurrent.SynchronousQueue$TransferStack@70298d52
	at sun.misc.Unsafe.park(Native Method)
	-  waiting on java.util.concurrent.SynchronousQueue$TransferStack@70298d52
	at java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:215)
	at java.util.concurrent.SynchronousQueue$TransferStack.awaitFulfill(SynchronousQueue.java:460)
	at java.util.concurrent.SynchronousQueue$TransferStack.transfer(SynchronousQueue.java:362)
	at java.util.concurrent.SynchronousQueue.poll(SynchronousQueue.java:941)
	at java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1073)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	...

"Downloader 1" Id=76 TIMED_WAITING on java.util.concurrent.SynchronousQueue$TransferStack@70298d52
	at sun.misc.Unsafe.park(Native Method)
	-  waiting on java.util.concurrent.SynchronousQueue$TransferStack@70298d52
	at java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:215)
	at java.util.concurrent.SynchronousQueue$TransferStack.awaitFulfill(SynchronousQueue.java:460)
	at java.util.concurrent.SynchronousQueue$TransferStack.transfer(SynchronousQueue.java:362)
	at java.util.concurrent.SynchronousQueue.poll(SynchronousQueue.java:941)
	at java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1073)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	...

"Downloader 0" Id=75 TIMED_WAITING on java.util.concurrent.SynchronousQueue$TransferStack@70298d52
	at sun.misc.Unsafe.park(Native Method)
	-  waiting on java.util.concurrent.SynchronousQueue$TransferStack@70298d52
	at java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:215)
	at java.util.concurrent.SynchronousQueue$TransferStack.awaitFulfill(SynchronousQueue.java:460)
	at java.util.concurrent.SynchronousQueue$TransferStack.transfer(SynchronousQueue.java:362)
	at java.util.concurrent.SynchronousQueue.poll(SynchronousQueue.java:941)
	at java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1073)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	...

"Netty Epoll Server IO #15" Id=74 RUNNABLE (in native)
	at io.netty.channel.epoll.Native.epollWait0(Native Method)
	at io.netty.channel.epoll.Native.epollWait(Native.java:117)
	at io.netty.channel.epoll.EpollEventLoop.epollWait(EpollEventLoop.java:223)
	at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:247)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:858)
	at java.lang.Thread.run(Thread.java:748)

"Netty Epoll Server IO #14" Id=73 RUNNABLE (in native)
	at io.netty.channel.epoll.Native.epollWait0(Native Method)
	at io.netty.channel.epoll.Native.epollWait(Native.java:117)
	at io.netty.channel.epoll.EpollEventLoop.epollWait(EpollEventLoop.java:223)
	at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:247)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:858)
	at java.lang.Thread.run(Thread.java:748)

"Netty Epoll Server IO #13" Id=72 RUNNABLE (in native)
	at io.netty.channel.epoll.Native.epollWait0(Native Method)
	at io.netty.channel.epoll.Native.epollWait(Native.java:117)
	at io.netty.channel.epoll.EpollEventLoop.epollWait(EpollEventLoop.java:223)
	at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:247)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:858)
	at java.lang.Thread.run(Thread.java:748)

"Netty Epoll Server IO #12" Id=71 RUNNABLE
	at io.netty.channel.epoll.Native.epollWait0(Native Method)
	at io.netty.channel.epoll.Native.epollWait(Native.java:117)
	at io.netty.channel.epoll.EpollEventLoop.epollWait(EpollEventLoop.java:223)
	at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:247)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:858)
	at java.lang.Thread.run(Thread.java:748)

"Netty Epoll Server IO #11" Id=70 RUNNABLE (in native)
	at io.netty.channel.epoll.Native.epollWait0(Native Method)
	at io.netty.channel.epoll.Native.epollWait(Native.java:117)
	at io.netty.channel.epoll.EpollEventLoop.epollWait(EpollEventLoop.java:223)
	at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:247)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:858)
	at java.lang.Thread.run(Thread.java:748)

"Netty Epoll Server IO #10" Id=68 RUNNABLE (in native)
	at io.netty.channel.epoll.Native.epollWait0(Native Method)
	at io.netty.channel.epoll.Native.epollWait(Native.java:117)
	at io.netty.channel.epoll.EpollEventLoop.epollWait(EpollEventLoop.java:223)
	at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:247)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:858)
	at java.lang.Thread.run(Thread.java:748)

"Netty Epoll Server IO #9" Id=67 RUNNABLE (in native)
	at io.netty.channel.epoll.Native.epollWait0(Native Method)
	at io.netty.channel.epoll.Native.epollWait(Native.java:117)
	at io.netty.channel.epoll.EpollEventLoop.epollWait(EpollEventLoop.java:223)
	at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:247)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:858)
	at java.lang.Thread.run(Thread.java:748)

"Netty Epoll Server IO #8" Id=66 RUNNABLE
	at io.netty.channel.epoll.Native.epollWait0(Native Method)
	at io.netty.channel.epoll.Native.epollWait(Native.java:117)
	at io.netty.channel.epoll.EpollEventLoop.epollWait(EpollEventLoop.java:223)
	at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:247)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:858)
	at java.lang.Thread.run(Thread.java:748)

"Netty Epoll Server IO #7" Id=65 RUNNABLE (in native)
	at io.netty.channel.epoll.Native.epollWait0(Native Method)
	at io.netty.channel.epoll.Native.epollWait(Native.java:117)
	at io.netty.channel.epoll.EpollEventLoop.epollWait(EpollEventLoop.java:223)
	at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:247)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:858)
	at java.lang.Thread.run(Thread.java:748)

"Netty Epoll Server IO #6" Id=64 RUNNABLE (in native)
	at io.netty.channel.epoll.Native.epollWait0(Native Method)
	at io.netty.channel.epoll.Native.epollWait(Native.java:117)
	at io.netty.channel.epoll.EpollEventLoop.epollWait(EpollEventLoop.java:223)
	at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:247)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:858)
	at java.lang.Thread.run(Thread.java:748)

"pool-3-thread-1" Id=63 WAITING on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@7eee7d27
	at sun.misc.Unsafe.park(Native Method)
	-  waiting on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@7eee7d27
	at java.util.concurrent.locks.LockSupport.park(LockSupport.java:175)
	at java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:2039)
	at java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:442)
	at java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1074)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)

"pool-5-thread-1" Id=62 WAITING on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@7ad1e622
	at sun.misc.Unsafe.park(Native Method)
	-  waiting on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@7ad1e622
	at java.util.concurrent.locks.LockSupport.park(LockSupport.java:175)
	at java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:2039)
	at java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:1081)
	at java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:809)
	at java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1074)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	...

"Chunk I/O Executor Thread-1" Id=60 WAITING on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@3fd3771c
	at sun.misc.Unsafe.park(Native Method)
	-  waiting on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@3fd3771c
	at java.util.concurrent.locks.LockSupport.park(LockSupport.java:175)
	at java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:2039)
	at java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:442)
	at java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1074)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)

"Netty Epoll Server IO #5" Id=56 RUNNABLE
	at io.netty.channel.epoll.Native.epollWait0(Native Method)
	at io.netty.channel.epoll.Native.epollWait(Native.java:117)
	at io.netty.channel.epoll.EpollEventLoop.epollWait(EpollEventLoop.java:223)
	at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:247)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:858)
	at java.lang.Thread.run(Thread.java:748)

"Netty Epoll Server IO #4" Id=55 RUNNABLE (in native)
	at io.netty.channel.epoll.Native.epollWait0(Native Method)
	at io.netty.channel.epoll.Native.epollWait(Native.java:117)
	at io.netty.channel.epoll.EpollEventLoop.epollWait(EpollEventLoop.java:223)
	at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:247)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:858)
	at java.lang.Thread.run(Thread.java:748)

"Server Watchdog" Id=51 RUNNABLE
	at sun.management.ThreadImpl.dumpThreads0(Native Method)
	at sun.management.ThreadImpl.dumpAllThreads(ThreadImpl.java:454)
	at net.minecraft.server.dedicated.ServerHangWatchdog.run(ServerHangWatchdog.java:45)
	at java.lang.Thread.run(Thread.java:748)

"Netty Epoll Server IO #3" Id=50 RUNNABLE
	at io.netty.channel.epoll.Native.epollWait0(Native Method)
	at io.netty.channel.epoll.Native.epollWait(Native.java:117)
	at io.netty.channel.epoll.EpollEventLoop.epollWait(EpollEventLoop.java:223)
	at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:247)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:858)
	at java.lang.Thread.run(Thread.java:748)

"File IO Thread" Id=45 TIMED_WAITING
	at java.lang.Thread.sleep(Native Method)
	at net.minecraft.world.storage.ThreadedFileIOBase.func_75736_b(SourceFile:51)
	at net.minecraft.world.storage.ThreadedFileIOBase.run(SourceFile:30)
	at java.lang.Thread.run(Thread.java:748)

"Netty Epoll Server IO #2" Id=42 RUNNABLE (in native)
	at io.netty.channel.epoll.Native.epollWait0(Native Method)
	at io.netty.channel.epoll.Native.epollWait(Native.java:117)
	at io.netty.channel.epoll.EpollEventLoop.epollWait(EpollEventLoop.java:223)
	at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:247)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:858)
	at java.lang.Thread.run(Thread.java:748)

"threadDeathWatcher-2-1" Id=33 TIMED_WAITING
	at java.lang.Thread.sleep(Native Method)
	at io.netty.util.ThreadDeathWatcher$Watcher.run(ThreadDeathWatcher.java:152)
	at io.netty.util.concurrent.DefaultThreadFactory$DefaultRunnableDecorator.run(DefaultThreadFactory.java:144)
	at java.lang.Thread.run(Thread.java:748)

"Netty Epoll Server IO #1" Id=32 RUNNABLE (in native)
	at io.netty.channel.epoll.Native.epollWait0(Native Method)
	at io.netty.channel.epoll.Native.epollWait(Native.java:117)
	at io.netty.channel.epoll.EpollEventLoop.epollWait(EpollEventLoop.java:223)
	at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:247)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:858)
	at java.lang.Thread.run(Thread.java:748)

"Netty Epoll Server IO #0" Id=31 RUNNABLE (in native)
	at io.netty.channel.epoll.Native.epollWait0(Native Method)
	at io.netty.channel.epoll.Native.epollWait(Native.java:117)
	at io.netty.channel.epoll.EpollEventLoop.epollWait(EpollEventLoop.java:223)
	at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:247)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:858)
	at java.lang.Thread.run(Thread.java:748)

"Server console handler" Id=22 RUNNABLE (in native)
	at java.io.FileInputStream.readBytes(Native Method)
	at java.io.FileInputStream.read(FileInputStream.java:255)
	at java.io.BufferedInputStream.read1(BufferedInputStream.java:284)
	at java.io.BufferedInputStream.read(BufferedInputStream.java:345)
	-  locked java.io.BufferedInputStream@3ac3ea9
	at sun.nio.cs.StreamDecoder.readBytes(StreamDecoder.java:284)
	at sun.nio.cs.StreamDecoder.implRead(StreamDecoder.java:326)
	at sun.nio.cs.StreamDecoder.read(StreamDecoder.java:178)
	-  locked java.io.InputStreamReader@e266cbe
	at java.io.InputStreamReader.read(InputStreamReader.java:184)
	...

"DestroyJavaVM" Id=21 RUNNABLE

"Server thread" Id=19 RUNNABLE
	at net.minecraft.world.chunk.BlockStateContainer.func_186013_a(BlockStateContainer.java:84)
	at net.minecraft.world.chunk.storage.ExtendedBlockStorage.func_177484_a(ExtendedBlockStorage.java:63)
	at net.minecraft.world.chunk.Chunk.<init>(Chunk.java:120)
	at net.minecraft.world.gen.ChunkGeneratorOverworld.func_185932_a(ChunkGeneratorOverworld.java:252)
	at net.minecraft.world.gen.ChunkProviderServer.func_186025_d(ChunkProviderServer.java:143)
	at net.minecraft.world.World.func_72964_e(World.java:310)
	at net.minecraft.world.ChunkCache.<init>(ChunkCache.java:37)
	at net.minecraft.pathfinding.PathNavigate.func_179680_a(SourceFile:113)
	...

"Server Infinisleeper" Id=18 TIMED_WAITING
	at java.lang.Thread.sleep(Native Method)
	at net.minecraft.server.dedicated.DedicatedServer$1.run(DedicatedServer.java:82)

"Snooper Timer" Id=17 TIMED_WAITING on java.util.TaskQueue@7bb37bae
	at java.lang.Object.wait(Native Method)
	-  waiting on java.util.TaskQueue@7bb37bae
	at java.util.TimerThread.mainLoop(Timer.java:552)
	at java.util.TimerThread.run(Timer.java:505)

"Thread-1" Id=16 WAITING on java.util.LinkedList@5ffc8853
	at java.lang.Object.wait(Native Method)
	-  waiting on java.util.LinkedList@5ffc8853
	at java.lang.Object.wait(Object.java:502)
	at com.zeitheron.hammercore.lib.zlib.utils.TaskedThread.run(TaskedThread.java:39)

"Signal Dispatcher" Id=4 RUNNABLE

"Finalizer" Id=3 WAITING on java.lang.ref.ReferenceQueue$Lock@3d47ac73
	at java.lang.Object.wait(Native Method)
	-  waiting on java.lang.ref.ReferenceQueue$Lock@3d47ac73
	at java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:143)
	at java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:164)
	at java.lang.ref.Finalizer$FinalizerThread.run(Finalizer.java:212)

"Reference Handler" Id=2 WAITING on java.lang.ref.Reference$Lock@3964f9
	at java.lang.Object.wait(Native Method)
	-  waiting on java.lang.ref.Reference$Lock@3964f9
	at java.lang.Object.wait(Object.java:502)
	at java.lang.ref.Reference.tryHandlePending(Reference.java:191)
	at java.lang.ref.Reference$ReferenceHandler.run(Reference.java:153)

Stacktrace:
	at net.minecraft.server.dedicated.ServerHangWatchdog.run(ServerHangWatchdog.java:62)
	at java.lang.Thread.run(Thread.java:748)

-- System Details --
Details:
	Minecraft Version: 1.12.2
	Operating System: Linux (amd64) version 3.10.0-862.14.4.el7.x86_64
	Java Version: 1.8.0_171, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 1636119600 bytes (1560 MB) / 3054501888 bytes (2913 MB) up to 3054501888 bytes (2913 MB)
	JVM Flags: 2 total; -Xmx3072M -Xms3072M
	IntCache: cache: 1, tcache: 1, allocated: 12, tallocated: 94
	FML: MCP 9.42 Powered by Forge 14.23.5.2768 86 mods loaded, 85 mods active
	States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored

	| State     | ID                                   | Version                | Source                                                   | Signature                                |
	|:--------- |:------------------------------------ |:---------------------- |:-------------------------------------------------------- |:---------------------------------------- |
	| UCHIJAAAA | minecraft                            | 1.12.2                 | minecraft.jar                                            | None                                     |
	| UCHIJAAAA | mcp                                  | 9.42                   | minecraft.jar                                            | None                                     |
	| UCHIJAAAA | FML                                  | 8.0.99.99              | forge-1.12.2-14.23.5.2768-universal.jar                  | e3c3d50c7c986df74c645c0ac54639741c90a557 |
	| UCHIJAAAA | forge                                | 14.23.5.2768           | forge-1.12.2-14.23.5.2768-universal.jar                  | e3c3d50c7c986df74c645c0ac54639741c90a557 |
	| UCHIJAAAA | creativecoredummy                    | 1.0.0                  | minecraft.jar                                            | None                                     |
	| UCHIJAAAA | passableleavescore                   | 1.0.0                  | minecraft.jar                                            | None                                     |
	| UCHIJAAAA | additionalbanners                    | 1.1.76                 | AdditionalBanners-1.12.2-1.1.76.jar                      | d476d1b22b218a10d845928d1665d45fce301b27 |
	| UCHIJAAAA | alcatrazcore                         | 1.0.3                  | alcatrazcore-1.0.3.jar                                   | 3c2d6be715971d1ed58a028cdb3fae72987fc934 |
	| UCHIJAAAA | backstab                             | 1.0.3                  | backstab-1.12.2-1.0.3.jar                                | None                                     |
	| UCHIJAAAA | badmobs                              | 1.1.36                 | BadMobs-1.12.2-1.1.36.jar                                | d476d1b22b218a10d845928d1665d45fce301b27 |
	| UCHIJAAAA | baubles                              | 1.5.2                  | Baubles-1.12-1.5.2.jar                                   | None                                     |
	| UCHIJAAAA | betteranimalsplus                    | 4.1.0                  | betteranimalsplus-4.1.0.jar                              | None                                     |
	| UCHIJAAAA | bettercombatmod                      | 1.5.6                  | BetterCombat-1.12.2-1.5.6.jar                            | None                                     |
	| UCHIJAAAA | bionisation3                         | 1.0.6                  | Bionisation3-1.12.2-1.0.6.jar                            | None                                     |
	| UCHIJAAAA | chargeblade                          | 1.0                    | chargeblade-1.0.jar                                      | None                                     |
	| UCHIJAAAA | redstoneflux                         | 2.1.0                  | RedstoneFlux-1.12-2.1.0.6-universal.jar                  | 8a6abf2cb9e141b866580d369ba6548732eff25f |
	| UCHIJAAAA | cofhcore                             | 4.6.1                  | CoFHCore-1.12.2-4.6.1.22-universal.jar                   | None                                     |
	| UCHIJAAAA | crafttweaker                         | 4.1.14                 | CraftTweaker2-1.12-4.1.14.jar                            | None                                     |
	| UCHIJAAAA | mantle                               | 1.12-1.3.3.39          | Mantle-1.12-1.3.3.39.jar                                 | None                                     |
	| UCHIJAAAA | jei                                  | 4.14.4.261             | jei_1.12.2-4.14.4.261.jar                                | None                                     |
	| UCHIJAAAA | tconstruct                           | 1.12.2-2.12.0.115      | TConstruct-1.12.2-2.12.0.115.jar                         | None                                     |
	| UCHIJAAAA | conarm                               | 1.2.3                  | conarm-1.12.2-1.2.3.jar                                  | 5d5b8aee896a4f5ea3f3114784742662a67ad32f |
	| UCHIJAAAA | cosmeticarmorreworked                | 1.12.2-v4              | CosmeticArmorReworked-1.12.2-v4.jar                      | aaaf83332a11df02406e9f266b1b65c1306f0f76 |
	| UCHIJAAAA | cosmeticarmorreworked|tombmanygraves | 1.12.2-v4              | CosmeticArmorReworked-1.12.2-v4.jar                      | aaaf83332a11df02406e9f266b1b65c1306f0f76 |
	| UCHIJAAAA | crafttweakerjei                      | 2.0.2                  | CraftTweaker2-1.12-4.1.14.jar                            | None                                     |
	| UCHIJAAAA | creativecore                         | 1.9.9                  | CreativeCore_v1.9.37_mc1.12.2.jar                        | None                                     |
	| UCHIJAAAA | cyberware                            | 0.2.10.15              | cyberware-1.12.2-0.2.10.15.jar                           | 67a0e286dc0d4b502f3c92ac20b953517b52d0a9 |
	| UCHIJAAAA | cyclopscore                          | 1.0.1                  | CyclopsCore-1.12.2-1.0.1.jar                             | bd0353b3e8a2810d60dd584e256e364bc3bedd44 |
	| UCHIJAAAA | dawnoftime                           | 0.1                    | dawnoftime-alpha-1.0.0.jar                               | None                                     |
	| UCHIJAAAA | dynamicstealth                       | 1.12.2.043             | DynamicStealth-1.12.2.043.jar                            | None                                     |
	| UCHIJAAAA | emoticons                            | 0.2.1                  | emoticons-0.2.1-1.12.2.jar                               | None                                     |
	| UCHIJAAAA | epicsiegemod                         | 13.163                 | EpicSiegeMod-13.163.jar                                  | None                                     |
	| UCHIJAAAA | everlastingabilities                 | 1.3.19                 | EverlastingAbilities-1.12.2-1.3.19.jar                   | bd0353b3e8a2810d60dd584e256e364bc3bedd44 |
	| UCHIJAAAA | fastleafdecay                        | v14                    | FastLeafDecay-v14.jar                                    | None                                     |
	| UCHIJAAAA | ftblib                               | 5.4.0.73               | FTBLib-5.4.0.73.jar                                      | None                                     |
	| UCHIJAAAA | ftbutilities                         | 5.3.0.68               | FTBUtilities-5.3.0.68.jar                                | None                                     |
	| UCHIJAAAA | gravestone                           | 1.10.2                 | gravestone-1.10.2.jar                                    | None                                     |
	| UCHIJAAAA | guideapi                             | 1.12-2.1.8-63          | Guide-API-1.12-2.1.8-63.jar                              | None                                     |
	| UCHIJAAAA | hammercore                           | 2.0.4.1                | HammerCore-1.12.2-2.0.4.1.jar                            | 4d7b29cd19124e986da685107d16ce4b49bc0a97 |
	| UCHIJAAAA | lucraftcore                          | 1.12.2-2.3.4           | LucraftCore-1.12.2-2.3.4.jar                             | None                                     |
	| UCHIJAAAA | heroesexpansion                      | 1.12.2-1.2.3           | HeroesExpansion-1.12.2-1.2.3.jar                         | None                                     |
	| UCHIJAAAA | howlingmoon                          | 0.9                    | howlingmooncustom.jar                                    | None                                     |
	| UCHIJAAAA | iblis                                | 0.5.2                  | iblis-0.5.2.jar                                          | None                                     |
	| UCHIJAAAA | iblis_headshots                      | 1.2.0                  | iblis-0.5.2.jar                                          | None                                     |
	| UCHIJAAAA | ichunutil                            | 7.1.4                  | iChunUtil-1.12.2-7.1.4.jar                               | None                                     |
	| UCHIJAAAA | improvableskills                     | 3.6.0r                 | ImprovableSkills-1.12.2-3.6.0r.jar                       | 4d7b29cd19124e986da685107d16ce4b49bc0a97 |
	| UCHIJAAAA | improvedbackpacks                    | 1.12.2-1.4.0.0         | ImprovedBackpacks-1.12.2-1.4.0.0.jar                     | None                                     |
	| UCHIJAAAA | infernalmobs                         | 1.7.5                  | InfernalMobs-1.12.2.jar                                  | None                                     |
	| UCHIJAAAA | invertedredstone                     | 0.1                    | invertedredstone-0.1.jar                                 | None                                     |
	| UCHIJAAAA | jepb                                 | 1.2.1                  | jepb-1.12-1.2.1.jar                                      | None                                     |
	| UCHIJAAAA | kindredlegacy                        | 1.12.2-1.2.1           | kindredlegacy-1.12.2-1.2.1.jar                           | None                                     |
	| UCHIJAAAA | livingenchantment                    | 3.1.5                  | living_enchantment-3.1.5.jar                             | None                                     |
	| UCHIJAAAA | locks                                | 2.2                    | Locks-2.2.jar                                            | None                                     |
	| UCHIJAAAA | radixcore                            | 1.12.x-2.2.1           | RadixCore-1.12.x-2.2.1-universal.jar                     | None                                     |
	| UCHIJAAAA | mca                                  | 1.12.2-5.3.1           | MCA-1.12.x-5.3.1-universal.jar                           | None                                     |
	| UCHIJAAAA | metallurgy                           | 0.0.5                  | Metallurgy-4-Reforged-0.0.5.jar                          | None                                     |
	| UCHIJAAAA | minecraftfuture                      | 0.0.8                  | minecraftfuture-0.0.8.jar                                | None                                     |
	| UCHIJAAAA | targeting_api                        | 0.12                   | targeting_api-0.12.jar                                   | None                                     |
	| UCHIJAAAA | mkultra                              | 0.80                   | MKUltra-0.80.jar                                         | None                                     |
	| UCHIJAAAA | mobamputation                        | 7.0.0                  | MobAmputation-1.12.2-7.0.0.jar                           | None                                     |
	| UCHIJAAAA | moreplayermodels                     | 1.12.2                 | MorePlayerModels_1.12.2(18jan18).jar                     | None                                     |
	| UCHIJAAAA | mtlib                                | 3.0.5                  | MTLib-3.0.5.jar                                          | None                                     |
	| UCHIJAAAA | collapsingcolumns                    | @VERSION@              | nonerdpoles-1.12.2-2.0.4.jar                             | None                                     |
	| UCHIJAAAA | opframe                              | 1.4.0                  | OnlinePicFrame_v1.4.23_mc1.12.2.jar                      | None                                     |
	| UCHIJAAAA | parry                                | 1.0                    | parry-1.0-hotfix.jar                                     | None                                     |
	| UCHIJAAAA | ra                                   | 1.12.2-1.0.0           | Recycle+Armor-1.12.2-1.2.1.jar                           | None                                     |
	| UCHIJAAAA | redstonepaste                        | 1.7.5                  | redstonepaste-mc1.12-1.7.5.jar                           | None                                     |
	| UCHIJAAAA | rewired                              | 1.12.2                 | ReWIRED-1.0.4.jar                                        | None                                     |
	| UCHIJAAAA | subcommonlib                         | 1.2.0.1                | subcommonlib-sublib-1.12.2-1.2.0.4.jar                   | None                                     |
	| UCHIJAAAA | rpginventory                         | 1.12.2                 | rpginventory-1.12-5.3.0.0.jar                            | None                                     |
	| UCHIJAAAA | special_ai                           | 1.1.3                  | Special_AI_v1.0.jar                                      | None                                     |
	| UCHIJAAAA | stg                                  | 1.12.2-1.2.3           | stg-1.12.2-1.2.3.jar                                     | None                                     |
	| UCHIJAAAA | tails                                | 1.12-1.10.0            | Tails-1.12-1.10.0.jar                                    | None                                     |
	| UCHIJAAAA | tiffitlib                            | 1.0.3                  | tiffitlib-1.0.3.jar                                      | None                                     |
	| UCHIJAAAA | tinkerscompendium                    | 1.4.2.22.1             | tinkersdefense-1.4.2.22.1.jar                            | None                                     |
	| UCHIJAAAA | tinkersarsenal                       | 0.5                    | tinkersarsenal-0.5.jar                                   | None                                     |
	| UCHIJAAAA | toroquest                            | 1.12.2-5.3             | toroquest-1.12.2-5.3.jar                                 | None                                     |
	| UCHIJAAAA | unhealthydying                       | 1.2.2.1                | Unhealthy-Dying-1.2.2.1.jar                              | None                                     |
	| UCHIJAAAA | vampire                              | 0.0.2B                 | vampire-0.0.2B.jar                                       | None                                     |
	| UCHIJAAAA | vampirism                            | 1.4.9                  | Vampirism-1.12.2-1.4.9.jar                               | None                                     |
	| UCHIJAAAA | teamlapen-lib                        | 1.4.9                  | Vampirism-1.12.2-1.4.9.jar                               | None                                     |
	| UCHIJAAAA | vampirism_integrations               | vampirism_integrations | VampirismIntegrations-1.12.2-1.1.0-alpha+12-09-22-44.jar | None                                     |
	| UCHIJAAAA | walljump                             | 1.12.2-1.2.1           | walljump-1.12.2-1.2.1.jar                                | None                                     |
	| UCHIJAAAA | worleycaves                          | 1.3.0                  | worleycaves-1.3.0.jar                                    | None                                     |
	| UCHIJAAAA | passableleaves                       | 2.0.0                  | PassableLeaves-1.12.2-2.0.0.jar                          | None                                     |
	| UD        | emoticons_core                       | 0.2.1                  | minecraft.jar                                            | None                                     |

	Loaded coremods (and transformers): 
PassableLeavesCore (PassableLeavesCore-1.12.2-1.0.0.jar)
  teamrtg.passableleaves.asm.PassableLeavesTransformer
NWRTweak (redstonepaste-mc1.12-1.7.5.jar)
  net.fybertech.nwr.NWRTransformer
LucraftCoreCoreMod (LucraftCore-1.12.2-2.3.4.jar)
  lucraft.mods.lucraftcore.core.LCTransformer
CreativePatchingLoader (CreativeCore_v1.9.37_mc1.12.2.jar)
  
EmoticonsCore (emoticons-0.2.1-1.12.2.jar)
  mchorse.emoticons.core.EmoCoreClassTransformer
HCASM (HammerCore-1.12.2-2.0.4.1.jar)
  com.zeitheron.hammercore.asm.HammerCoreTransformer
	Pulsar/tconstruct loaded Pulses: 
		- TinkerCommons (Enabled/Forced)
		- TinkerWorld (Enabled/Not Forced)
		- TinkerTools (Enabled/Not Forced)
		- TinkerHarvestTools (Enabled/Forced)
		- TinkerMeleeWeapons (Enabled/Forced)
		- TinkerRangedWeapons (Enabled/Forced)
		- TinkerModifiers (Enabled/Forced)
		- TinkerSmeltery (Enabled/Not Forced)
		- TinkerGadgets (Enabled/Not Forced)
		- TinkerOredict (Enabled/Forced)
		- TinkerIntegration (Enabled/Forced)
		- TinkerFluids (Enabled/Forced)
		- TinkerMaterials (Enabled/Forced)
		- TinkerModelRegister (Enabled/Forced)

	Hammer Core Information: FluidDictionary Info: Registered 2 fluids.
OreDictionary Info: Registered 2461 names; 1049 stacks.
Configuration Info: IyBDb25maWd1cmF0aW9uIGZpbGUNCg0KY2xpZW50IHsNCiAgICAjIFNob3VsZCBIYW1tZXIgQ29yZSByZXBsYWNlIGVuY2hhbnRtZW50IGdsaW50IGNvbG9yIHdpdGggbW9yZSBtYXRjaGluZyBjb2xvcnMgZm9yIHNvbWUgdmFuaWxsYSBpdGVtcz8gW2RlZmF1bHQ6IHRydWVdDQogICAgQjoiQ3VzdG9tIEVuY2hhbnRtZW50IENvbG9ycyBmb3IgVmFuaWxsYSBJdGVtcyI9dHJ1ZQ0KDQogICAgIyBTaG91bGQgSGFtbWVyIENvcmUgcG9ydGZvcndhcmQgeW91ciBsb2NhbCB3b3JsZCB3aGVuIHlvdSBzaGFyZSB3aXRoIGl0IHRvIExBTj8NCiAgICAjIFRoaXMgZmVhdHVyZSBkb2Vzbid0IHdvcmsgb24gYWxsIGNsaWVudHMsIHlvdSBhcmUgZ29pbmcgdG8gZ2V0IGEgbWVzc2FnZSBpbiBjaGF0IGlmIGl0IHdhcyBzdWNjZXNzZnVsL3Vuc3VjY2Vzc2Z1bCBbZGVmYXVsdDogdHJ1ZV0NCiAgICBCOiJMQU4gVVBuUCI9dHJ1ZQ0KDQogICAgIyBSZXBsYWNlIHZhbmlsbGEgZnVybmFjZSBhbmQgYnJld2luZyBzdGFuZCBwcm9ncmVzcyBiYXJzIHRvIHVzZSBzbW9vdGggcmVuZGVyaW5nPyBbZGVmYXVsdDogdHJ1ZV0NCiAgICBCOiJTbW9vdGggVmFuaWxsYSBHdWlzIj10cnVlDQp9DQoNCg0KaXdyZW5jaCB7DQogICAgIyBXaGF0IGlzIHRoZSBjb2xvciB3aGVuIHRoZSBob3ZlcmVkIGJsb2NrIGlzIHdyZW5jaGFibGU/IChjb2xvciBlbmNvZGVkIGluIGhleCBmb3JtIHN1Y2ggYXMgUlJHR0JCKQ0KICAgIFM6R3JlZW49MjJGRjIyDQoNCiAgICAjIFdoYXQgaXMgdGhlIGNvbG9yIHdoZW4gdGhlIGhvdmVyZWQgYmxvY2sgaXMgTk9UIHdyZW5jaGFibGU/IChjb2xvciBlbmNvZGVkIGluIGhleCBmb3JtIHN1Y2ggYXMgUlJHR0JCKQ0KICAgIFM6UmVkPUZGMjIyMg0KfQ0KDQoNCiJ2YW5pbGxhIGltcHJvdmVtZW50cyIgew0KICAgICMgU2hvdWxkIEhhbW1lciBDb3JlIGZvcmNlLXNwYXduIEVuZGVyIERyYWdvbiBFZ2cgb24gRW5kZXIgRHJhZ29uIGRlYXRoPyBbZGVmYXVsdDogdHJ1ZV0NCiAgICBCOiJBbHdheXMgU3Bhd24gRHJhZ29uIEVnZyI9dHJ1ZQ0KfQ0KDQoNCg==
Pastebin Connection: Okay.
Dependent Mods:
    -Improvable Skills (improvableskills) @3.6.0r

	Profiler Position: N/A (disabled)
	Player Count: 1 / 20; [EntityPlayerMP['starwolf62'/715, l='world', x=31.48, y=74.49, z=330.47]]
	Is Modded: Definitely; Server brand changed to 'fml,forge'
	Type: Dedicated Server (map_server.txt)

**Comment by Laike-Endaril** [comment]
First time seeing this type of error.  Is it reproducible?

**Comment by rfwainug** [comment]
I suppose I'll try and see.

**Comment by rfwainug** [comment]
I've failed to reproduce it, so I assume it's in conjunction with MCA as per these lines "at net.minecraft.entity.ai.EntityAITasks.func_75774_a(SourceFile:94)
at net.minecraft.entity.EntityLiving.func_70626_be(EntityLiving.java:763)
at net.minecraft.entity.EntityLivingBase.func_70636_d(EntityLivingBase.java:2352)
at net.minecraft.entity.EntityLiving.func_70636_d(EntityLiving.java:577)
at net.minecraft.entity.EntityAgeable.func_70636_d(EntityAgeable.java:178)" being very identical to MCA characteristics, specifically the ageable tag seeing as feral wolves do not have children.

**Comment by rfwainug** [comment]
I'll give that a try and see if I come up with anything different.

**Comment by Laike-Endaril** [comment]
I can try to run some tests as well eventually, but right now I'm in the middle of adding some new features.  I'll check this thread when I'm done with the features and then see if I can figure anything out.

**Comment by rfwainug** [comment]
Unfortunately I'm hitting a wall. It seems that mca is just naturally crashing my client from the server and I'm fairly certain your mod has nothing to do with it.

**Comment by rfwainug** [comment]
Well, it's working again and I've tried reproducing. I'm stumped. Guess this ones' on hold until I know more about it.

**Comment by Laike-Endaril** [comment]
You know what?  My brother and I set up a server with my own indev modpack not long ago and I now I remember something similar happened last night.  I'm not running MCA *or* better animals (though I'm considering adding better animals), so there's that...

I still have no idea of the cause, but it may very well be a DS thing.  It might make the thread verbose, but post every crash you get for this if you can.  In this case, the more info the better.

**Comment by Laike-Endaril** [comment]
So after some googling around, I found this issue rearing its head in several places, none of them involving DS.  Someone mentioned removing client-side mods and allocating more RAM, which led me to think about the java garbage collector cycle and take a closer look at your log...

`Memory: 1636119600 bytes (1560 MB) / 3054501888 bytes (2913 MB) up to 3054501888 bytes (2913 MB)
JVM Flags: 2 total; -Xmx3072M -Xms3072M`

You might want to increase the allocated RAM a bit, because if the garbage collector has a near-death experience and takes forever due to low RAM, that would trigger this error.  To add to the confusion, the 1560MB/2913MB shown in the report would be the status of the RAM *after* the GC recovers, which means the lag spike has already occurred and the watchdog gives up even though it just recovered.  Basically, the status *before* the GC issue would probably have been 2913/2913, or very close to it.



tl;dr

* Not the direct fault of any mod (unless one has a bad memory leak)

* Best solution: allocate more RAM and maybe remove any client-side-only mods from the server

* 2nd best solution: increase max tick setting in Minecraft's server.properties file, or set it to -1 (-1 means disable the tick time check).  This will keep the watchdog from shutting the server down, but it won't remove the lag which caused the watchdog to bite.

Someone should probably sticky this somewhere XD
